.github/workflows/ci.yml

```diff
@@ -85,7 +106,7 @@ jobs:
               exit 1;;
           esac
           cd sdist-test
-          bundle/build.sh "$bundle_os"
+          ci/fix-home bundle/build.sh "$bundle_os"
```

Comment:
Looks like this should be `../ci/fix-home` and is the reason why the alpha-03 release didn't build correctly.

Reply:
Oh, sorry. Thanks for fixing it.

Reply:
No worries! This was an easy thing to miss.